docs/en/Extending-Existing-Entities-Core.md

```diff
@@ -0,0 +1,232 @@
+
+    Add-Migration "Added_Address_To_User"
+
+This will create a new Entity Framework migration class:
+
+```csharp
+    public partial class Added_Address_To_User : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "Address",
+                table: "AbpUsers",
+                nullable: true);
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
+                name: "Address",
+                table: "AbpUsers");
+        }
+    }
+```
+
+Since it's automatically created, we don't have to know what it does for most cases. Now, we can update our database with this command:
+
+    Update-Database
+
+When we check **AbpUsers** table in the database, we can see the new **Address** field:
+
+<img src="images/extend-entities-user-address.png" alt="Address for Users" class="img-thumbnail" width="505" height="102" />
+
+For testing purposes, we can enter some data for existing users by hand.
+
+### Show Address On The UI
+
+Note: The UI part of this document is written for ASP.NET Core MVC & jQuery version of ASP.NET Zero. 
+
+**GetUsers** method in Authorization\\Users\\**UserAppService.cs** (in .Application project) is used for getting list of users by clients. It returns a list of **UserListDto** (we always use [DTOs](https://aspnetboilerplate.com/Pages/Documents/Data-Transfer-Objects) for client communication). So, we should add the Address property to UserListDto too:
+
+```csharp
+public class UserListDto : EntityDto<long>, IPassivable, IHasCreationTime
+{
+    //...existing code
+
+    public string Address { get; set; }
+}
+```
+
+Since UserListDto **auto maps** from User entity, no need to change **UserAppService.GetUsers** method. Now, we can go to UI side to add Address property to the **users table**. Open 
+.Web\\wwwroot\\view-resources\Areas\\{AppAreaName}\Views\\Users\\**Index.js** and add Address column right after CreationTime column:
+
+```javascript
+                },
+                {
+                    targets: 9,
+                    data: "creationTime",
+                    render: function (creationTime) {
+                        return moment(creationTime).format('L');
+                    }
+                },
+                {
+                    targets: 10,
+                    data: 'address'
+                }
+            ]
+        });
+```
+
+And in .Web\\Areas\\{AppAreaName}\Views\\Users\\**Index.cshtml**, add Address column:
+
+
+```html
+                            <th>@L("Active")</th>
+                            <th>@L("CreationTime")</th>
+                            <th>@L("Address")</th>
+                        </tr>
+                    </thead>
+                </table>
+```
+
+That's all. Let's run the application and open the **users page**:
```

Comment:
also add a note here for running "npm run create-bundles", so user can see the field on the UI.